### DISCUSSION TITLE
Chamilo for self-paced courses with self-assessment

### DISCUSSION THREAD

**Discussion by questcat** [chamilo/chamilo-lms#5373] [General]
Good day,

I am evaluating Chamilo for **self-paced courses** with mentors to assist only when needed.  Learners control their own time schedule. They also evaluate their own skill-levels based on given criteria.  This allows them to choose alternative learning paths to study a topic in more detail before embarking on tests that contribute to a grade. 

Is there a way to create a self-assessment form on Chamilo?  I tried using a test, but a self-assessment should not have any 'right' or 'wrong' answers.  A survey could have worked - but the learner needs instant feedback on what to do after the survey is done.  Also - I see no way to add a survey to a learning path?

Any other suggestions - maybe a plugin that I am missing?  My next option is to look at embedding html (maybe H5P or some other means).

Thank you,

**Comment by LudiscapeXApi** [comment]
Hello,

In your case, it's recommended to use one session per user and associate it with a survey, which will serve as a questionnaire in your scenario. This will allow you to collect self-assessment feedback from learners without requiring 'right' or 'wrong' answers. You can configure this survey to provide instant feedback to learners once they've completed it. This approach should meet your needs for self-assessment in Chamilo.

Damien
